verkle-trie/benches/benchmarks/proof_ipa.rs

```diff
@@ -0,0 +1,96 @@
+use ark_ff::{BigInteger256, UniformRand};
+use banderwagon::Fr;
+use criterion::{black_box, criterion_group, BenchmarkId, Criterion};
+use ipa_multipoint::{
+    crs::CRS,
+    lagrange_basis::{LagrangeBasis, PrecomputedWeights},
+    multiproof::{MultiPoint, ProverQuery},
+    transcript::Transcript,
+};
+use rand::Rng;
+use rayon::prelude::IntoParallelIterator;
+use rayon::prelude::*;
+
+const VECTOR_WIDTH :usize = 256;
+
+fn multiproof(c: &mut Criterion) {
```

Comment:
Benchmarking multiproof, which can be thought of (Aggregating polynomials + single IPA), so kind of an e2e bench.
We benchmark for different setups regarding number of aggregated polys.